### PR TITLE
adding myself

### PR DESCRIPTION
**Pull request by fuddl**
@b-g how about this?

### CODE REVIEW

**Comment by b-g** [comment]
@fuddl Many thanks for this! That works ... I guess the rather special mouth is on purpose and should stay?

**Comment by fuddl** [comment]
Screenshot_20200525-131103

File: about/index.html [hunk 1]
```diff
@@ -1239,6 +1239,21 @@ <h3>Kris Kowal</h3>
...
+              <div class="teammember">
+                <div><a href="../library#author=Alexander%20Müller"><img class="lazy"
+                      data-src="../img/team/alexander-mueller.svg"></a></div>
+                <!--image-->
+                <h3>Alexander Müller</h3>
+                <div class="team">
+                  <p>Emoji design and some Code</p>
+                </div>
+                <ul>
+                  <li><a class="iconlink" rel="noopener" href="https://twitter.com/kriskowal" target="_blank"><img
```
Comment (b-g) [inline about/index.html:1251]: Wrong links ... please fix

File: about/index.html [hunk 2]
```diff
@@ -1239,6 +1239,21 @@ <h3>Kris Kowal</h3>
...
+                        alt="twitter" src="../img/brand-icons/twitter.svg"></a></li>
+                  <li><a class="iconlink" rel="noopener" href="https://github.com/kriskowal" target="_blank"><img alt="github"
```
Comment (b-g) [inline about/index.html:1253]: Wrong link ... please fix

**Comment by b-g** [comment]
@fuddl OK + I see ... :) Last thing ... all your social media links are pointing to a different user. Please add yours. Will merge after.

**Comment by b-g** [comment]
@fuddl Many thanks + merged. Will go online with the next deployment ... give it a few days.

**Pull request merged**